code/control/SAMLController.php

```diff
@@ -90,7 +90,11 @@ public function acs()
                 continue;
             }
 
-            $member->$field = $attributes[$claim][0];
+            if(is_array($attributes[$claim]) && count($attributes[$claim]) > 1) {
```

Comment:
`is_array` here is redundant as `$attributes[$claim]` is always expected to be an array (as per the `else`)